src/components/TopBar/TopBar.vue

```diff
@@ -56,16 +56,27 @@
 				</div>
 			</span>
 
-			<NcButton
-				:aria-label="threadNotificationLabel"
-				:title="threadNotificationLabel"
-				:primary="!!threadNotification"
-				@click="() => {}">
+			<NcActions
+				:aria-label="t('spreed', 'Subscribe to thread')"
+				:title="t('spreed', 'Subscribe to thread')"
+				:variant="threadNotificationVariant">
```

Comment:
Doesn't it miss a capability check?

Reply:
It's inside of `<div v-if="currentThread" />`, so I don't think it's needed